iocore/eventsystem/I_Thread.h

```diff
@@ -145,8 +145,7 @@ class Thread
   Thread &operator=(const Thread &);
 
 public:
-  ink_thread start(const char *name, size_t stacksize = DEFAULT_STACKSIZE, ThreadFunction f = NULL, void *a = NULL,
-                   void *stack = NULL);
+  ink_thread start(const char *name, size_t stacksize, ThreadFunction f, void *a, void *stack);
```

Comment:
Consider removing ``ThreadFunction`` and making ``Thread::execute()`` pure virtual since ``ThreadFunction`` is never used anywhere.